src/components/Dashboard/FilingHistoryList/filings/ConsentContinuationOut.vue

```diff
@@ -8,10 +8,11 @@
       <div>
         <p
           v-if="expiry && !isConsentExpired"
-          class="mt-4"
+          class="mt-0"
```

Comment:
Before:

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/241912ce-db51-49fa-8bcf-e97ea532f82b)

After:

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/4babacd4-24be-4cbe-808f-6c452db07799)

Reply:
Note that the screenshots above are out of date -- now we don't display the order details (the paragraph "Six Month Consent to Continue Out" and "a detail").